lib/dataspec/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "rpcperf-dataspec"
+version = "0.1.0"
+edition = "2021"
+
```

Comment:
Should we inherit some of the metadata from the top-level manifest? See Pelikan repo for a reference on what we're doing there. Things like, license and all that. Versioning should remain independent.

Reply:
Ack, makes sense, thanks for the pointer